pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -133,16 +133,54 @@ message NamespaceOption {
     optional bool host_ipc = 3;
 }
 
+// PodSecurityContext holds pod-level security configuration that will be applied to a container.
+message PodSecurityContext {
+    // The configurations for the sandbox's namespaces.
+    // This will be used only if the PodSandbox uses namespace for isolation.
+    optional NamespaceOption namespace_options = 2;
```

Comment:
GRPC will generate `ThingOptions` based on `thing_options`, so document it as `ThingOptions` to not break godoc conventions.